src/main/resources/swagger/api-docs.yaml

```diff
@@ -1558,21 +1558,22 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
-  /api/resources/v2/filter:
+  /api/resources/v2/list:
```

Comment:
nit: I don't think the verb (filter/list) is required - it looks like all the rest of the Sam endpoints are nouns, with the HTTP action representing the move being made. Here `GET /api/resources/v2` pretty clearly connotes a 'list' action.

Reply:
I'm testing out actually combining the two endpoints into one. It looks like I actually can do some polymorphism with the responses in Swagger!